ecosystem_tests/ecosystem_tests_cli/constants.py

```diff
@@ -29,3 +29,4 @@
 DEFAULT_LICENSE_PATH = 'license.yaml'
 DEFAULT_CONTAINER_NAME = 'cfy_manager'
 DEFAULT_UNINSTALL_ON_SUCCESS = True
+DEFAULT_DIRECTORY_PATH = './'
```

Comment:
Maybe we should change it to.....

```suggestion
DEFAULT_DIRECTORY_PATH = '$PWD'
```